docs/design/data_safety.md

```diff
@@ -0,0 +1,153 @@
+Data Safety in ICU4X
+====================
+
+At its core, ICU4X is a set of algorithms that map input data to human-ready outputs. Since a core value proposition of ICU4X is the ability to dynamically load data, we can't always assume that our data are valid and pristine. This doc covers how we think about this problem and handle it in ICU4X.
+
+## Resolution
+
+1. _Whereas_ it is rare to be 100% confident about the safety of your data\*; and
```

Comment:
I find this `Whereas <statement>; and` style extremely annoying to read. I have never seen this before, where is this from? What's wrong with `<statement>`?

Reply:
https://www.google.com/search?q=wheras+resolved

Reply:
So...because legal documents are written like this? We should prioritize readability over legal tradition.

Reply:
I find the syntax to be a readable way to lay out an argument that has many partially-related pieces of evidence and conclusions; it allows people like Henri to point out objections to specific clauses in isolation, for example. But I can try to improve the readability.

Reply:
You can still have a list of statements, just please drop the "whereas", "resolved that" and "; and"s.